src/main/interface/cli.c

```diff
@@ -3271,7 +3271,11 @@ STATIC_UNIT_TESTED void cliGet(char *cmdline)
     const clivalue_t *val;
     int matchedCommands = 0;
 
+    uint8_t backupPidProfile = getCurrentPidProfileIndex();
```

Comment:
I had a look at this, and I am wondering if it wouldn't be better if we pulled the profile resetting (and LED_STRIP re-evaluating) out of `resetConfigs` (which is only really used in `cli.c`), and put it into an `activateConfigs()` together with the rest of the activation / validation that is done in `resetEEPROM()`? I will open a counter-proposal with my idea of doing this.

Reply:
I'm fine with that but note that `resetConfigs()` is also called by `resetEEPROM()` in fc/config.c

Reply:
I know. But part of my suggestion sit to reshuffle functionality in `resetEEPROM()`.